lib/sqlalchemy/dialects/mysql/mariadb.py

```diff
@@ -5,15 +5,24 @@
 # This module is part of SQLAlchemy and is released under
 # the MIT License: https://www.opensource.org/licenses/mit-license.php
 # mypy: ignore-errors
-from .base import MariaDBIdentifierPreparer
+from .base import MariaDBIdentifierPreparer, MySQLTypeCompiler
```

Comment:
this will fail our pep8 checks, install [pre-commit](https://pre-commit.com/) so that commits will format everything when you do a commit:

```
pre-commit install
```

Reply:
thanks, I'll update this